mermaid-init.js

```diff
@@ -0,0 +1 @@
+mermaid.initialize({startOnLoad:true});
```

Comment:
Denne fila og `mermaid.min.js` er kopiert fra Mermaid-integrasjonen til mdbook-mermaid og er påkrevd for å få diagrammene til å fungere.